src/components/facets/FacetRow.tsx

```diff
@@ -28,6 +36,7 @@ export class FacetRow extends React.Component<IFacetRowProps, any> {
           checked={this.props.isChecked}
           onClick={() => this.props.onToggleFacet(this.props.facetRow)}>
           <Label classes={['label']}>{label}</Label>
+          {!_.isUndefined(this.props.facetRow.count) ? <span className='facet-value-count'>{this.props.facetRow.count}</span> : null}
```

Comment:
This is more of a personal preference, but I would move the ternary logic into a `const` above the return. That's just what I've seen elsewhere in the repo.